keyboards/keebio/iris/keymaps/mojitas/keymap.c

```diff
@@ -0,0 +1,238 @@
+#include QMK_KEYBOARD_H
+#include "keymap_swedish.h"
+
+//Heavily modified keymap. Some features:
+//Multiple layouts, I use dvorak as main.
+//Nordic(swedish) signs
+//Symbols, numpad, arrows/navigation reachable under the alpas via the layers
+//Mirrored ctl, alt and shift to be able to use both hands when doing commands
+//Gaming layer, qwerty with space on left half.
+
+enum layer_names {
+    _DVORAK,
+    _COLEMAK,
+    _QWERTY,
+    _WORKMAN,
+    _GAMING,
+    _NUMPAD,
+    _LOWER,
+    _RAISE,
+    _ADJUST
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  DVORAK,
+  COLEMAK,
+  WORKMAN,
+  GAMING,
+  NUMPAD,
+};
+
+#define KC_ KC_TRNS
+
+#define KC_Sw2 RALT(KC_2) // Nordic @
+#define KC_Sw3 RALT(KC_3) // Nordic something
+#define KC_Sw4 RALT(KC_4) // Nordic something
+#define KC_Sw5 RALT(KC_5) // Nordic something
+#define KC_Sw6 RALT(KC_6) // ...
+#define KC_Sw7 RALT(KC_7)
+#define KC_Sw8 RALT(KC_8)
+#define KC_Sw9 RALT(KC_9)
+#define KC_Sw0 RALT(KC_0)
+#define KC_Tild RALT(KC_RBRC)
+#define KC_Bsls RALT(KC_MINS)
+#define KC_Bar RALT(KC_NUBS)
+#define KC_Less S(KC_NUBS)
```

Comment:
There are still all of these that can be converted to `SE_` keycodes directly in the keymap.